src/components/NavBar/NavBar.vue

```diff
@@ -46,8 +52,37 @@
 				EventBus.$emit('overlay-closeVisibile');
 			});
 		},
+
+		mounted() {	
+			this.$nextTick(() => {
+			window.addEventListener('resize', this.onResize);
+			})
+		},
+		methods: {
+			onResize(){
+				if(!this.isMobile()){
+					this.visible = true;
+				} else {
+					this.visible = false;
+					EventBus.$emit('overlay-closeVisibile');
+				}
```

Comment:
This is a good start, however we should also look at laptops or split screens or small monitors. So we should check the width of the window. Depending on width and/or aspect ratio we should determine if it should always be visible or not.

So I would suggest to make a function that looks something like this example. *This example is just some thought of mind, not saying it should look/act like this for sure...*
```js
alwaysVisible() {
	if(this.isMobile()) {
		return false;
	}
	var width, height = ... // get these vars somehow
	if(width/height > ...) {
		return true;
	}
	if(width > ...) {
		return true;
	}
	return false;
}
```

Reply:
Yes, in my first idea was to use only the aspect ratio of the screen, but if you put your smartphone the horizontal, the navbar will be fixed, and that would be another problem. So i choose to detect if its on mobile.

The best solution is indeed to consider the device, aspect ratio and resolution, luckly its a easy fix.